tests/dummy/app/index.html

```diff
@@ -14,6 +14,16 @@
     <link rel="stylesheet" href="assets/vendor.css">
     <link rel="stylesheet" href="assets/dummy.css">
 
+    <script type="text/javascript"
+        src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.6.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
+    </script>
+    <script type="text/x-mathjax-config">
```

Comment:
This is not a dependency for consuming apps - not all apps are going to need mathjax.  For most of my work on this component, i was using the dummy app to test that things were working; in this case, I wanted to see mathjax in the search results.